databox/docs/basics/crawler.md

```diff
@@ -0,0 +1,67 @@
+# Crawler
+
+A Crawler is a specialized [Functino](./function.md) that allows you to scrape a webpage and automatically cache results. The following is a simple example:
+
+```js
+import { Crawler, HeroFunctionPlugin } from '@ulixee/databox-plugins-hero';
+
+const crawler = new Crawler(async context => {
+  const hero = await new context.Hero();
+  await hero.goto('https://ulixee.org');
+
+  const resources = await hero.activeTab.waitForResources({ url: 'index.json' });
+  for (const resource of resources) await resource.$addToDetachedResources('xhr');
+  return hero;
+}, HeroFunctionPlugin);
+```
+
+Crawlers automatically create a local [Table] to cache results. This means you can re-use the last Crawl by default. A desired "age" can be specified by including a `maxTimeInCache` parameter. Omitting will cause a new crawl to initiate.
+
+```js
+const crawl = await crawler.stream({ input: { maxTimeInCache: 60 } });
+const heroReplay = await HeroReplay(crawl);
```

Comment:
Is `craw` a array or object? The documentation below makes it seem like it's an array, but this code makes it seem like an object.

Reply:
Should be an array

Reply:
Does HeroReplay accept an array?

Reply:
No, should be 
```js
const [crawl] = await crawler.stream({ input: { maxTimeInCache: 60 } });
const heroReplay = await HeroReplay(crawl);
```